wandb/cli/cli.py

```diff
@@ -848,6 +860,7 @@ def _parse_settings(settings):
         or env.get("WANDB_ENTITY")
         or config.get("entity")
         or api.settings("entity")
+        or api.default_entity
```

Comment:
Slightly risky change... 
The old way, the entity was resolved at the time of the sweep being created, it is actually done in the upsert sweep mutation.

This new way is doing a query from the public api and getting hopefully the same information.... in the future this change is worth noting in the review. 

This is probably fine since this is in the CLI and not the python interface it should be ok as this isnt very optimized and there is no offline mode here.

Reply:
Removed.